js/public/bindings.js

```diff
@@ -41,25 +44,26 @@ const setBinding = (e, values) => {
         delete bindings[oldId[1]][oldId[2]]
     }
     binding.id = `bind-${type}-${idx}`
-
     bindings[type][idx] = {
         name: getVal('name'),
         invert: getVal('invert') === 'true',
-        action: getVal('action'),
+        action: getVal('action')
```

Comment:
Added linter config, please `npm run lint` unless your editor is already doing it.
Trailing commas used here to enable lighter diff's and easier copy/pasting